arangod/Aql/ExecutionNode.h

```diff
@@ -1237,6 +1237,9 @@ class MaterializeRocksDBNode : public MaterializeNode,
   ExecutionNode* clone(ExecutionPlan* plan, bool withDependencies,
                        bool withProperties) const override final;
 
+  bool alwaysCopiesRows() const override { return false; }
+  bool isIncreaseDepth() const override { return false; }
```

Comment:
Do I understand right it should be false because block is passthrough and we call nextDataRow at the same cycle with advanceRow?
And if it's not true, like for MultiGet, it can be true?

Reply:
Hmm, I also not fully understand purpose of `isIncreaseDepth`.

It looks like best case is
`alwaysCopiesRows` -- true // because it's mean we can reuse registers
`isIncreaseDepth` -- false // we don't increase depth